cloudsync/event.py

```diff
@@ -338,8 +338,6 @@ def _fill_event_path(self, event: Event):
         state = self.state.lookup_oid(self.side, event.oid)
         if state:
             event.path = state[self.side].path
-        if not event.path and event.exists in (True, None):
-            self._make_event_accurate(event)
```

Comment:
`_fill_event_path` is overloaded in the `SmartEventManager`

Reply:
removed this block in the smartsync codepath also.

let me know if you think it is still needed there for other reasons.

Reply:
Now I'm reading the code below, and it seems like you added a pass at finding and updating inaccurate sync entries already? So this code might be fine as-is, if it accomplishes what I noted in the last comment